types/field_type.go

```diff
@@ -103,6 +103,19 @@ func AggFieldType(tps []*FieldType) *FieldType {
 	return &currType
 }
 
+// TryToFixFlenOfDatetime try to fix flen of Datetime for specific func or other field merge cases
+func TryToFixFlenOfDatetime(resultTp *FieldType) bool {
```

Comment:
Why do we need to return a bool value even if we do not check it ?

Reply:
removed